nucliadb_core/src/metrics/metric/tokio_runtime.rs

```diff
@@ -19,39 +19,55 @@
 
 use std::sync::{Mutex, MutexGuard, TryLockError};
 
+use crate::tracing::info;
 use prometheus_client::metrics::counter::Counter;
 use prometheus_client::metrics::gauge::Gauge;
 use prometheus_client::metrics::histogram::Histogram;
 use prometheus_client::registry::Registry;
+use tokio;
 use tokio::runtime::Handle;
 use tokio_metrics::{RuntimeIntervals, RuntimeMonitor};
 
 use crate::{node_error, NodeResult};
 
 pub struct TokioRuntimeObserver {
-    runtime: Handle,
-    intervals: Mutex<RuntimeIntervals>,
+    runtime: Option<Handle>,
+    intervals: Option<Mutex<RuntimeIntervals>>,
     metrics: TokioRuntimeMetrics,
 }
 
 impl TokioRuntimeObserver {
     pub fn new(registry: &mut Registry) -> Self {
-        let runtime = Handle::current();
-        let monitor = RuntimeMonitor::new(&runtime);
-        // We need to store RuntimeIntervals iterator instead of RuntimeMonitor to
-        // get incremental values. We need a Mutex to be Send and Sync (for our
-        // Meter users)
-        let intervals = Mutex::new(monitor.intervals());
-        Self {
-            runtime,
-            intervals,
-            metrics: TokioRuntimeMetrics::new(registry),
+        let runtime = Handle::try_current();
+        if runtime.is_err() {
+            info!("Cannot export tokio runtime metrics, no runtime available");
+            Self {
+                runtime: None,
+                intervals: None,
+                metrics: TokioRuntimeMetrics::new(registry),
+            }
+        } else {
+            let runtime = runtime.unwrap();
+            let monitor = RuntimeMonitor::new(&runtime);
+            // We need to store RuntimeIntervals iterator instead of RuntimeMonitor to
+            // get incremental values. We need a Mutex to be Send and Sync (for our
+            // Meter users)
+            let intervals = Mutex::new(monitor.intervals());
+            Self {
+                runtime: Some(runtime),
+                intervals: Some(intervals),
+                metrics: TokioRuntimeMetrics::new(registry),
+            }
         }
```

Comment:
```rust
let Ok(runtime) =  Handle::try_current() else {
          info!("Cannot export tokio runtime metrics, no runtime available");
           return Self {
                runtime: None,
                intervals: None,
                metrics: TokioRuntimeMetrics::new(registry),
            };
};
let monitor = RuntimeMonitor::new(&runtime);
// We need to store RuntimeIntervals iterator instead of RuntimeMonitor to
// get incremental values. We need a Mutex to be Send and Sync (for our
// Meter users)
let intervals = Mutex::new(monitor.intervals());
Self {
       runtime: Some(runtime),
       intervals: Some(intervals),
       metrics: TokioRuntimeMetrics::new(registry),
}
```

Reply:
Ah dang, thanks. I actually forgot about these changes. It was quite annoying that I couldn't run these tests locally without the changes.